e2e_tests/database/test_database.json

```diff
@@ -376,6 +376,19 @@
     "change_message": "[{\"changed\": {\"fields\": [\"Bewertung Verwaltung\", \"Bewertung Klimaaktionsplan\"]}}]"
   }
 },
+{
```

Comment:
To prevent this file from becoming longer and longer, it might be a good idea to remove these changes before commit.